sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -163,9 +169,10 @@ function getRequestBody(body?: unknown, contentType: string = ""): RequestBody {
 
   switch (firstType) {
     case "multipart/form-data":
-      return isRLCFormDataInput(body)
-        ? { formData: processFormData(body) }
-        : { body: JSON.stringify(body) };
+      if (Array.isArray(body)) {
+        return { multipartBody: buildMultipartBody(body as PartDescriptor[]) };
+      }
+      return { body: JSON.stringify(body) };
```

Comment:
let's have a seperated issue to talk about this, maybe we could have a common way to handle un-expected.

Reply:
Yeah, this is just what we were doing before, I don't think it's good behavior.